pkg/core/consensus/reduction/firststep/step.go

```diff
@@ -94,14 +94,21 @@ func (p *Phase) Run(ctx context.Context, queue *consensus.Queue, evChan chan mes
 	}
 
 	p.handler = reduction.NewHandler(p.Keys, r.P, r.Seed)
-	// first we send our own Selection
-	if p.handler.AmMember(r.Round, step) {
-		m, _ := p.SendReduction(r.Round, step, &p.selectionResult.Candidate)
 
-		// Queue my own vote to be registered locally
-		evChan <- m
+	// send our own Selection
+	var wg sync.WaitGroup
```

Comment:
Any reason to use a `WaitGroup` without waiting for it?

Reply:
It's part of the input params for  AsyncSend.Go, but not using it anymore. So, I'll remove them.